index.html

```diff
@@ -3356,6 +3356,31 @@ <h3><em>RTL/bidi text (TBD)</em></h3>
 
 
 
+
+
+
+
+<section id="typ_box_posn" class="subtopic">
+<h3>Setting box positioning coordinates when text direction varies</h3>
+
+  	<div class="req" id="vertical_box_posn">
+	<p class="advisement">Box positioning coordinates must take into account whether the text is horizontal or vertical.</p>
+	</div>
+
+<p>It is typical, when localizing a user interface or web page, to create mirror-images for the RTL and LTR versions. For example, it is likely that a box that appears near the left side of a window containing English content would appear near the right side of the window if the content is Arabic or Hebrew. It should preferably automatic for this to change, based on the base direction of the current context, unless there is a strong reason for using absolute geometry. One way to achieve this is to use keywords such as <code class="kw" translate="no">start</code> and <code class="kw" translate="no">end</code>, rather than <code class="kw" translate="no">left</code> and <code class="kw" translate="no">right</code>, to indicate position.</p>
```

Comment:
Link "RTL" and "LTR" to their definitions. I would reverse the order of these in the first sentence.

---

"It should preferably automatic" => "It should preferably _be_ automatic"